cmd/openfga/main_test.go

```diff
@@ -1562,15 +1677,23 @@ func GRPCReadAuthorizationModelTest(t *testing.T, tester OpenFGATester) {
 
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
-			response, err := client.ReadAuthorizationModel(context.Background(), test.input)
+			if test.testData != nil {
+				modelResp, err := client.WriteAuthorizationModel(context.Background(), &openfgav1.WriteAuthorizationModelRequest{
+					StoreId:         test.input.StoreId,
+					SchemaVersion:   typesystem.SchemaVersion1_1,
+					TypeDefinitions: parser.MustTransformDSLToProto(test.testData.model).TypeDefinitions,
+				})
+				test.input.Id = modelResp.AuthorizationModelId
+				require.NoError(t, err)
+			}
+			_, err := client.ReadAuthorizationModel(context.Background(), test.input)
 
 			s, ok := status.FromError(err)
 			require.True(t, ok)
 			require.Equal(t, test.output.errorCode.String(), s.Code().String())
 
-			if test.output.errorCode == codes.OK {
-				_ = response // use response for assertions
-				// require.Equal(t, test.output.resp.Allowed, response.Allowed)
+			if s.Code() == codes.OK {
+				require.NoError(t, err)
```

Comment:
for the != codes.OK case, should we assert require.Error?

Reply:
You mean assert on the exact error message? Yes we can do that, but at this moment i don't really care about those, only the response status code (as a first pass).

We can submit a follow up PR to assert on all the error messages

Reply:
No, what I meant is at least that we are returning err

Reply:
Oh! Let me try